src/CommunityToolkit.Maui/MauiCommunityToolkitOptions.cs

```diff
@@ -0,0 +1,28 @@
+using CommunityToolkit.Maui.Behaviors;
+using CommunityToolkit.Maui.Converters;
+
+namespace CommunityToolkit.Maui;
+
+/// <summary>
+/// .NET MAUI Community Toolkit Options.
+/// </summary>
+public class MauiCommunityToolkitOptions
```

Comment:
After typing the class name out a few times here, it's kind of repetitive. 

What do you think about renaming it to `public class Options`? 

Then we'll have the following two classes: 
- `CommunityToolkit.Maui.Options` 
- `CommunityToolkit.Maui.Core.Options`